packages/fxa-settings/src/components/Settings/FlowRecoveryKeyConfirmPwd/index.test.tsx

```diff
@@ -257,148 +254,4 @@ describe('FlowRecoveryKeyConfirmPwd with Change Action', () => {
     screen.getByRole('button', { name: 'Create new account recovery key' });
     screen.getByRole('link', { name: 'Cancel' });
   });
-
-  it('disables the submit button when the input is empty', async () => {
-    renderFlowPage(accountWithSuccess, RecoveryKeyAction.Change);
```

Comment:
I'd removed them because it felt like duplication since the RecoveryKeyAction is no longer passed in as a prop, but I can see that we'd want to ensure that all these tests pass for both actions anyway. Reverted/updated to include the second set of tests.